src/site/content/en/metrics/lcp/index.md

```diff
@@ -102,9 +102,10 @@ considered for Largest Contentful Paint are:
 - [Block-level](https://developer.mozilla.org/docs/Web/HTML/Block-level_elements)
   elements containing text nodes or other inline-level text elements children.
 
-Note, restricting the elements to this limited set was intentional in order to
-keep things simple in the beginning. Additional elements (like the full `<svg>` support or
-`<video>` elements without poster images) may be added in the future as more research is conducted.
+Additionally, the following elements will be considered for a limited set of users as of version 116 of Chrome, but will be considered for all users as of version 117:
+
+- The first frame painted for `<video>` elements, negating the need for the `poster` attribute to used for autoplaying video
```

Comment:
I thought these were going to be listed in the main set of elements with an asterisk pointing to a note explaining that the change is being rolled out partially.

Reply:
A bit of difference in format, but the preceding paragraph reads:

> Additionally, the following elements will be considered **for a limited set of users as of version 116 of Chrome**, but **will be considered for all users as of version 117**:

It may not entirely accurate as is, but willing to workshop it as needed.

Reply:
To give a bit more information on my intent with this approach, I think it's worth setting the additional elements in a different list until the changes are applicable across 100% of all Chrome installs (or close enough to it).

Reply:
Similar to @philipwalton's suggestion, WDYT about adding these to the list above with `(as of [August 2023](changelog))` at the end of each item? I think that would also address the question of whether the change is tied to any particular Chrome release.